### ISSUE TITLE
undefined is not a function

### ISSUE THREAD

**Issue by TheRealRed7** [robinpowered/react-native-audio-jack#5]
Probably my mistake as I am new to React Native but..

I am just trying to display the AudioJack status but my test app is throwing this error:

**TypeError: undefined is not a function (evaluating '_reactNativeAudioJack2.default.isPluggedIn()')**

I have altered the AndroidManifest to look like this:  [https://pastebin.com/vQJTU2vy](https://pastebin.com/vQJTU2vy)

`(manifest location: node_modules\react-native-audio-jack\android\src\main)`

and my test app looks like this:  [https://pastebin.com/3T5Ypaum](https://pastebin.com/3T5Ypaum)

Let me know if you think I should be asking this question on stackoverflow as I'm not sure if its an issue with react-native-audio-jack or my scrappy code.

Thanks

**Comment by yinghang** [comment]
Can you please confirm that you actually did run the `react-native link react-native-audio-jack` command after installation? The error seems to suggest that RN is not detecting the native module.

Also, you're modifying the wrong `AndroidManifest.xml` file, you shouldn't be changing the manifest within the module. Instead, you should be editing your RN project's manifest file.

**Comment by TheRealRed7** [comment]
Hi, yes I did run **react-native link react-native-audio-jack**.

The response was:  Scanning folders for symlinks in C:\Users\Red7\audiojacktest\node_modules (59ms)

I have removed my edit to the incorrect androidmanifest file. I test and build in Expo so I dont think I get the andoidmanifest file that you are referring to unless I export the project?

Does my code run for you? I am beginning to think this is a code error rather than an issue with react-native-audio-jack.
